Transfer to the new repo.

LightnessOfBeing opened the pull request.

LightnessOfBeing committed:
Transfer to the new repo.

yurii-litvinov commented on MyZipFile/src/main/java/ru/spbau/mit/vishnyakov/MyZipFile.java:
```diff
@@ -0,0 +1,130 @@
+
+                    if (!entry.isDirectory() && entry.getName().matches(regExpression)) {
+                        write(zip.getInputStream(entry),
+                                new BufferedOutputStream(new FileOutputStream(
```
Плохая идея создавать их тут, если где-то внутри будет исключение, файл никто не закроет

yurii-litvinov replied:
Напоминаю

yurii-litvinov commented on MyZipFile/src/test/java/ru/spbau/mit/vishnyakov/MyZipFileTest.java:
```diff
@@ -0,0 +1,73 @@
+
+        File file1 = new File(directory, "aaa");
+        boolean check1 = file1.exists();
+        assertEquals(true, check1);
```
assertTrue?

yurii-litvinov commented on MyZipFile/src/test/java/ru/spbau/mit/vishnyakov/MyZipFileTest.java:
```diff
@@ -0,0 +1,73 @@
+
+        File file3 = new File(directory, "bbb");
+        boolean check3 = file3.exists();
+        assertEquals(false, check3);
```
assertFalse?

yurii-litvinov commented on MyZipFile/build.gradle:
```diff
@@ -0,0 +1,14 @@
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```
org.jetbrains.annotations нет, так что из консоли не собирается

yurii-litvinov reviewed (changes requested):
О, бинарники и сгенерённые файлы в гите, надо убрать

LightnessOfBeing committed:
Deleted unnecessary files.

LightnessOfBeing committed:
FIxed all complaints.

yurii-litvinov reviewed (changes requested):
Тут тоже надо всю папек build грохнуть

LightnessOfBeing committed:
Deleted build folder.

yurii-litvinov commented on MyZipFile/src/test/java/ru/spbau/mit/vishnyakov/MyZipFileTest.java:
```diff
@@ -0,0 +1,73 @@
+
+        File file1 = new File(directory, "aaa");
+        boolean check1 = file1.exists();
+        assertTrue(check1);
```
А зачем не `assertTrue(file1.exists());`?

LightnessOfBeing committed:
Added try-with-resource and deleted unnecessary boolean variables in test class.

yurii-litvinov commented on MyZipFile/src/main/java/ru/spbau/mit/vishnyakov/MyZipFile.java:
```diff
@@ -0,0 +1,126 @@
+            out.write(buffer, 0, len);
+        }
+        out.close();
+        in.close();
```
Вообще, не он их открыл --- не ему их и закрывать

yurii-litvinov reviewed (approved):
Ок, зачтена
